runners/gradle-plugin/src/main/kotlin/main.kt

```diff
@@ -298,6 +297,7 @@ open class DokkaTask : DefaultTask() {
 
     @TaskAction
     fun generate() {
+        project.configurations.getByName("dokkaRuntime").defaultDependencies{ dependencies -> dependencies.add(project.dependencies.create(dokkaFatJar)) }
```

Comment:
Looks like you should initialize property `dokkaRuntime` in case it is null and apply `defaultDependencies` to it